data/human/news.txt

```diff
@@ -2190,3 +2190,342 @@ news "graffiti on spaceport wall"
 			"Veni, vidi, hyperjumped away."
 			"Those who dance will be called fools by those who cannot hear the music."
 			"Better half a sentence, "
+
+news "High Tension Situation"
+	location
+		attributes "ccor"
+	name
+		word
+			"Standoff"
+	message
+		word
+			"On the corner "
+			"By a landing pad "
+			"At the entrance to the spaceport market, "
+		word
+			"some CCOR guerrillas are "
+			"there's a group of people in CCOR colors "
+		word
+			"distributing pamphlets"
+			"handing out literature"
+			"delivering aid"
+			"rallying against the local authorities"
+			"supporting local protests"
+		word
+			". "
+		word
+			"When "
+		word
+			"the planetary authorities "
+			"a gang of local pirates "
+		word
+			"accost "
+			"confront "
+		word
+			"them and demand that they "
+		word
+			"disperse"
+			"make themselves scarce"
+			"abandon their objective"
+			"shut up and buzz off"
```

Comment:
To be honest it sounds a bit silly for pirates/planetary authorities to be saying this.

Reply:
```suggestion
```